lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.spec.ts

```diff
@@ -314,25 +333,9 @@ describe('CardViewDateItemComponent', () => {
         });
     });
 
-    it('should be possible update a date-time', async () => {
```

Comment:
Why did you delete this test?

Reply:
hi @nikita-web-ua i have not deleted this test, just modified it's name and added it's functionality in updateDateTIme() function to avoid duplication of code.

Reply:
Because we have 2 cases here now one with `properties.cm:from` and other without `properties.cm:from`